lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -79,59 +132,63 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
     }
 
     // Subscribe to text and selection changes.
-    _subscriptions.add(domElement.onInput.listen(_handleChange));
-
-    _subscriptions.add(domElement.onKeyDown.listen(_maybeSendAction));
-
+    _subscriptions.add(activeDomElement.onInput.listen(_handleChange));
+    _subscriptions.add(activeDomElement.onKeyDown.listen(_maybeSendAction));
     _subscriptions.add(html.document.onSelectionChange.listen(_handleChange));
-
     preventDefaultForMouseEvents();
   }
 
-  @override
-  void initializeElementPlacement() {
-    // Element placement is done by [TextField].
-  }
-
   @override
   void initializeTextEditing(InputConfiguration inputConfig,
       {_OnChangeCallback? onChange, _OnActionCallback? onAction}) {
-    // In accesibilty mode, the user of this class is supposed to insert the
-    // [domElement] on their own. Let's make sure they did.
-    assert(html.document.body!.contains(domElement));
-
     isEnabled = true;
     _inputConfiguration = inputConfig;
     _onChange = onChange;
     _onAction = onAction;
     _applyConfiguration(inputConfig);
   }
 
-  @override
-  void setEditingState(EditingState? editingState) {
-    super.setEditingState(editingState);
-
-    // Refocus after setting editing state.
-    domElement.focus();
-  }
-
   @override
   void placeElement() {
     // If this text editing element is a part of an autofill group.
     if (hasAutofillGroup) {
       placeForm();
     }
-    domElement.focus();
+    activeDomElement.focus();
+  }
+
+  @override
+  void initializeElementPlacement() {
+    // Element placement is done by [TextField].
   }
 
   @override
```

Comment:
remove empty override?

Reply:
The empty override makes these methods noop.